go/store/nbs/table_persister.go

```diff
@@ -63,6 +63,9 @@ type tablePersister interface {
 type tableFilePersister interface {
 	tablePersister
 
+	// CopyTableFile copies the table file with the given fileId from the reader to the TableFileStore.
+	CopyTableFile(ctx context.Context, r io.ReadCloser, fileId string, chunkCount uint32) error
+
 	// Path returns the file system path.
 	Path() string
```

Comment:
i think we want to remove this right?

Reply:
We talked about this and I think decided not to remove it because it's used in remotesrv? https://github.com/dolthub/dolt/blob/main/go/libraries/doltcore/remotesrv/grpc.go#L127